src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -223,93 +213,82 @@ const InstancesListPage = (): JSX.Element => {
     [t]
   );
 
-  const customToolbarElement = (
-    <>
-      <Button
-        ouiaId="create-smart-event-instance"
-        onClick={(): void => setShowCreateInstance(true)}
-      >
-        {t("instance.createSEInstance")}
-      </Button>
-      <CreateInstance
-        isOpen={showCreateInstance}
-        onClose={(): void => setShowCreateInstance((prev) => !prev)}
-        getCloudProviders={getCloudProvidersWithRegions}
-        getSchema={getSchema}
-        createBridge={handleCreate}
-      />
-    </>
-  );
-
-  const onPaginationChange = useCallback(
-    (pageNumber: number, pageSize: number): void => {
-      const correctPageNumber =
-        pageSize === currentPageSize ? pageNumber : FIRST_PAGE;
-      setCurrentPage(correctPageNumber);
-      setCurrentPageSize(pageSize);
-      getBridges(correctPageNumber, pageSize);
-    },
-    [currentPageSize, getBridges]
-  );
-
   const rowOuiaId = useCallback(
-    (row): string | undefined => (row as BridgeResponse).name,
+    ({ row, rowIndex }): string =>
+      (row as BridgeResponse).name ?? `table-row-${String(rowIndex)}`,
     []
   );
+
   const renderActions = useCallback(
     ({ row, ActionsColumn }): JSX.Element => (
       <ActionsColumn items={tableActions(row as IRowData)} />
     ),
     [tableActions]
   );
+
+  const onCreateInstance = (): void => setShowCreateInstance(true);
+
+  const emptyStateNoData = (
+    <EmptyState
```

Comment:
It is our implementation we can add ouiaId here easily, but the create button is visible only if there is no data (bridges / processes).

Reply:
Exactly. This is the same observation I made in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/269#discussion_r1033967337